week-3/alarmclock/alarmclock.js

```diff
@@ -1,19 +1,70 @@
-function setAlarm() {}
 
-// DO NOT EDIT BELOW HERE
+let countdown;
 
-var audio = new Audio("alarmsound.mp3");
+function setAlarm() {
+  const input = document.getElementById("alarmSet");
+  const heading = document.getElementById("timeRemaining");
+
+  let timeInSeconds = parseInt(input.value, 10);
```

Comment:
The parseInt function doesn't need the second param to be specfied for this use case - have a look at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt to find out why